static/views/edit-developer-profile.html

```diff
@@ -5,6 +5,7 @@
 		<link href="/x/s/m/{{ .SiteLogoImageID }}?w=50&h=50" rel="shortcut icon">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<link rel="stylesheet" href="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.css">
+		<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
it takes 50ms and 8KB just to download an icon library, I didn't include fonteawesome from the beginning as I tried to keep only the strict necessary to keep the site loading snappy. can you please remove this and add simple boring 'add experience' or 'github' buttons where needed? I can help out to clarify how to display stuff if you are in doubt. we try not to add unnecessary external libraries

Reply:
Makes sense. removed.